TheModernIlluminati/TheModernIlluminati/Pages/Laureate.cshtml

```diff
@@ -19,19 +19,19 @@
                         </ol>
                         <div class="carousel-inner">
                             <div class="carousel-item" data-interval="2000">
-                                <img src="~/images/MicrosoftTeams-image (14).png" width="300" height="200">
+                                <img src="~/images/MicrosoftTeams-image (14).png" width="300" height="200" />
                             </div>
                             <div class="carousel-item" data-interval="2000">
-                                <img src="~/images/MicrosoftTeams-image (15).png" width="300" height="200">
+                                <img src="~/images/MicrosoftTeams-image (15).png" width="300" height="200" />
                             </div>
                             <div class="carousel-item active text-center" data-interval="2000">
-                                <img src="~/images/MicrosoftTeams-image (16).png" width="300" height="200">
+                                <img src="~/images/MicrosoftTeams-image (16).png" width="300" height="200" />
                             </div>
                             <div class="carousel-item" data-interval="2000">
-                                <img src="~/images/MicrosoftTeams-image (17).png" width="300" height="200">
+                                <img src="~/images/MicrosoftTeams-image (17).png" width="300" height="200" />
                             </div>
                             <div class="carousel-item" data-interval="2000">
-                                <img src="~/images/MicrosoftTeams-image (18).png" accesskeywidth="300" height="200">
+                                <img src="~/images/MicrosoftTeams-image (18).png" accesskeywidth="300" height="200" />
```

Comment:
I Corrected the HTML format by adding end tags for the image element. As per best practices, empty elements must always be closed.
https://www.w3schools.com/html/html_xhtml.asp

Reply:
This is a good find.  Also see notes above... there's other technical debt that needs to be addressed here.